spec/features/agreement_spec.rb

```diff
@@ -1,14 +1,12 @@
 require 'spec_helper'
 
-
 describe "Display agreement", :type => :feature do
-  before do
-    sign_in :admin
```

Comment:
This is important! You should always use the lowest amount of authorisation as possible. Do not test functionality as an admin when a basic user or no user can suffice. Heck, do you even need to be logged in to see the agreement page now that I think about it? Answer is No.